code/game/objects/items/devices/paicard.dm

```diff
@@ -168,6 +168,9 @@
 				src.add_overlay("pai-sunglasses")
 
 /obj/item/paicard/proc/alertUpdate()
+	var/image/I = image(icon, src, icon_state = "pai-alert")
+	flick_overlay_view(I, src, 5 SECONDS)  // overlay workaround
```

Comment:
How is this a workaround? I don't think this comment is necessary, otherwise PR is good

Reply:
i spent a whole hour trying to figure out how overlays worked

Reply:
This is fine, just remove the comment and I'll approve